jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/awsrules/federated/CheckCloudTrailMultiRegionEnabled.java

```diff
@@ -0,0 +1,111 @@
+ * use this file except in compliance with the License.  You may obtain a copy
+ * of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ ******************************************************************************/
+/**
+  Copyright (C) 2019 T Mobile Inc - All Rights Reserve
+  Purpose:
+  Author :Anukriti
+  Modified Date: Feb 27, 2019
+
+ **/
+package com.tmobile.cloud.awsrules.federated;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.MDC;
+
+import com.tmobile.cloud.awsrules.utils.PacmanUtils;
+import com.tmobile.cloud.constants.PacmanRuleConstants;
+import com.tmobile.pacman.commons.PacmanSdkConstants;
+import com.tmobile.pacman.commons.exception.InvalidInputException;
+import com.tmobile.pacman.commons.rule.Annotation;
+import com.tmobile.pacman.commons.rule.BaseRule;
+import com.tmobile.pacman.commons.rule.PacmanRule;
+import com.tmobile.pacman.commons.rule.RuleResult;
+
+@PacmanRule(key = "check-cloudtrail-multiRegion-enabled", desc = "This rule checks for AWS CloudTrail multi region enabled", severity = PacmanSdkConstants.SEV_MEDIUM, category = PacmanSdkConstants.SECURITY)
+public class CheckCloudTrailMultiRegionEnabled extends BaseRule {
+
+    private static final Logger logger = LoggerFactory
+            .getLogger(CheckCloudTrailMultiRegionEnabled.class);
+
+    /**
+     * The method will get triggered from Rule Engine with following parameters
+     *
+     * @param ruleParam
+     *
+     *            ************* Following are the Rule Parameters********* <br>
+     * <br>
+     *
+     *            ruleKey : check-for-aws-cloudtrail-config <br>
+     * <br>
+     *
+     *            severity : Enter the value of severity <br>
+     * <br>
+     *
+     *            ruleCategory : Enter the value of category <br>
+     * <br>
+     *
+     *            roleIdentifyingString : Configure it as role/pac_ro <br>
+     * <br>
+     *
+     * @param resourceAttributes
+     *            this is a resource in context which needs to be scanned this
+     *            is provided y execution engine
+     *
+     */
+
+    @Override
+    public RuleResult execute(Map<String, String> ruleParam,
+            Map<String, String> resourceAttributes) {
+        logger.debug("========CheckAWSCloudTrailConfig started=========");
+        Annotation annotation = null;
+        String cloudTrailInput = ruleParam.get("inputCloudTrailName");
+        String roleIdentifyingString = ruleParam
```

Comment:
This is needed only when if your creating the client and describing.So you can remove it

Reply:
All the review changes are incorporated.